### DISCUSSION TITLE
Is it possible to retrieve the Twitter username via Auth?

### DISCUSSION THREAD

**Discussion by dsumer** [supabase/supabase#2180] [Questions]
I want my users to be able to login via Twitter in my application backed by supabase Auth. And I also need to retrieve the Twitter username from those users. But in the user_metadata I only see the fullname and avatar URL of the user.

Is it possible to retrieve the Twitter username somehow? Cheers!

<img width="574" alt="CleanShot 2021-07-01 at 13 44 38@2x" src="https://user-images.githubusercontent.com/4418879/124119214-82c4e200-da72-11eb-945a-c79ddd8ff19f.png">

**Comment by silentworks** [comment]
You will probably need to create an issue for this on the issue tracker, but make sure you label it a feature as it would be nice to have this for sure.

**Reply by dsumer** [reply]
I've created a pull request: https://github.com/supabase/gotrue/pull/127
